chain/chain/src/runtime/metrics.rs

```diff
@@ -36,6 +36,48 @@ pub(crate) static PREPARE_TX_SIZE: Lazy<HistogramVec> = Lazy::new(|| {
     .unwrap()
 });
 
+pub(crate) static PREPARE_TX_GAS: Lazy<HistogramVec> = Lazy::new(|| {
+    try_create_histogram_vec(
+        "near_prepare_tx_gas",
+        "How much gas was spent for processing new transaction.",
+        &["shard_id"],
+        // 100e9 = 100 Ggas
+        // A transaction with no actions costs 108 Ggas to process.
+        // A typical function call costs ~300 Ggas.
+        // The absolute maximum is defined by `max_tx_gas` = 500 Tgas.
+        // This ranges from 100 Ggas to 409.6 Tgas as the last bucket boundary.
+        Some(exponential_buckets(100e9, 2.0, 12).unwrap()),
+    )
+    .unwrap()
+});
+
+pub(crate) static PREPARE_TX_REJECTED: Lazy<HistogramVec> = Lazy::new(|| {
+    try_create_histogram_vec(
+        "near_prepare_tx_rejected",
+        "The number of transactions removed from the transaction pool due to congestion.",
```

Comment:
```suggestion
        "The number of transactions rejected when producing a chunk.",
```

Reply:
At first I didn't realize that it was total number of transactions in a single chunk, so I was wondering why it's a histogram.